enterprise/cluster/src/main/java/org/neo4j/cluster/statemachine/StateTransitionLogger.java

```diff
@@ -79,6 +83,19 @@ public void stateTransition( StateTransition transition )
             Object payload = transition.getMessage().getPayload();
             if ( payload != null )
             {
+                if (payload instanceof Payload )
+                {
+                    try
+                    {
+                        payload = atomicBroadcastSerializer.receive( (Payload) payload);
+                    }
+                    catch ( Throwable e )
+                    {
+                        // Ignore
+                        e.printStackTrace();
```

Comment:
Are we sure printing the stack trace is the proper action here? If this is ignoreable, then nothing should be printed - if we are interested in this, then we should print int out in a more structured way?